torch/_subclasses/meta_utils.py

```diff
@@ -485,19 +485,28 @@ def _view_from_base(base, t):
                                 # fake_t -> fake_base via reverse view func
                                 fake_base = t._rev_view_func_unsafe(fake_t)
 
+                                # TODO: Clean up this mess wrt requires_grad / is_leaf!
                                 # fake_base -> fake_input_base
                                 # pull from cache if possible; detach and store in cache otherwise
                                 fake_input_base = self.get_tensor_memo(base)
                                 if fake_input_base is None:
-                                    # TODO: Fix this! the requires_grad and is_leaf statuses
-                                    # are wrong after the detach()
```

Comment:
nit: any reason the comment changed? (doesn't really matter, minor downside = git blaming for the comment is more annoying)